### PR TITLE
refactor(FOROME-1253): unify func modals with dialogs

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1253
In addition:
1. Unified select attribute modal and fixed disappeared scroll.

### CODE REVIEW

File: src/components/units-list/units-list.tsx [hunk 1]
```diff
@@ -55,19 +57,20 @@ export const UnitsList = ({
       className={cn(
         styles.unitsList,
         isDark && styles.unitsList_dark,
+        { 'pl-4': !isModal },
         className,
       )}
     >
       <FunctionalUnits
-        className={styles.unitsList__functional}
+        className={cn(styles.unitsList__functional, { 'pt-4': !isModal })}
```
Comment (QSdmitrioul) [inline src/components/units-list/units-list.tsx]: Add this style to CSS module: 
```
&unit-list {
  &__functional {
     &__modal {
       padding-top: 16px;
    }
  }
}
```

Comment (QSdmitrioul) [reply src/components/units-list/units-list.tsx]: In every place of this file

File: src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx [hunk 2]
```diff
@@ -49,16 +48,17 @@ export const ModalCompoundHet = observer((): ReactElement => {
...
-        groupName={attributeName}
-        handleClose={modalsVisibilityStore.closeModalCompoundHet}
-      />
-
+    <Dialog
+      isOpen={modalsVisibilityStore.isCompoundHetDialogVisible}
+      onClose={modalsVisibilityStore.closeCompoundHetDialog}
+      title={attributeName}
+      width="m"
+      actions={''}
```
Comment (QSdmitrioul) [inline src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx]: `actions=""`

Comment (QSdmitrioul) [reply src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx]: This should be applied in every place you use `Dialog`

File: src/components/units-list/units-list.tsx (hunk 1 above)
Comment (AlMaQntr) [reply src/components/units-list/units-list.tsx]: padding-top: theme("spacing.4")

File: src/components/units-list/units-list.tsx [hunk 3]
```diff
@@ -76,7 +79,11 @@ export const UnitsList = ({
       {subHeader && (
         <div className={styles.unitsList__subHeader}>{subHeader}</div>
       )}
-      <div className={styles.unitsList__list} id={listContainerId}>
+
+      <div
+        className={cn(styles.unitsList__list, { 'max-h-70': isModal })}
```
Comment (AlMaQntr) [inline src/components/units-list/units-list.tsx]: maybe `styles.unitsList__list_modal`

**Review by AlMaQntr — APPROVED** [review]

File: src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx (hunk 2 above)
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx]: That's some weird stuff here. Probably it's better to add a 'hideActions' prop to 'DialogCard'

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**